### ISSUE TITLE
Slack Import for one channel does not work because of "Missed import types: { thread_broadcast: ..."

### ISSUE THREAD

**Issue by svrnwnsch** [RocketChat/Rocket.Chat#12676]
### Description:

I cannot import one channel of my Slack export into Rocket.Chat.
The debug lock is telling me:
```
Slack Importer ➔ debug Slack is now at importer_importing_messages.
Meteor ➔ method getImportProgress -> userId: XXXXXXXXXXXX , arguments:  [ 'slack' ]
Missed import types: { thread_broadcast: 
   { text: 'XXXXXXXX',
     unfurl_links: true,
     client_msg_id: 'XXXXXXXXXX',
     unfurl_media: true,
     thread_ts: 'XXXXXXXXX',
     is_thread_broadcast: true,
     attachments: [ [Object] ],
     type: 'message',
     subtype: 'thread_broadcast',
     user: 'YYYYYYYYYY',
     root: 
      { type: 'message',
        user: 'YYYYYYYYY',
        text: 'XXXXXXXXXX',
        attachments: [Array],
        thread_ts: 'XXXXXXX',
        reply_count: 1,
        replies: [Array],
        unread_count: 1,
        ts: 'XXXXXXXX' },
     ts: 'XXXXXXXXX' } }
Slack Importer ➔ debug Slack is now at importer_finishing.

```
I guess that the subtype: thread_broadcast is not working?

I am running rocket.chat version 0.71.1

**Comment by engelgabriel** [comment]
> We have decided that was better to delay our v1 release and get the Threads done properly than get the feature out the way it was initially implemented on the develop branch. We have "rebranded" our current implementation of Threads to Discussions (was it creates a new room type) and will implement Threads exactly the same way as Slack does.
>
> In our vision there are 2 different use cases on how to break out of the main conversation: one very easy and organic, with a single click for small side talk, and other, much more structured, that creates a new conversation with its own life cycle.
>
> We have tried to have a single solution for both, and as a result, no one was happy with it.
So we have given up on having a single approach for both use cases and will implement both: the Slack-like industry standard and our brand new one, called Discussions.

https://github.com/RocketChat/Rocket.Chat/issues/13843

**Comment by engelgabriel** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/1112

**Comment by lpilz** [comment]
This should be closed by #17130
